packages/ember-routing/tests/system/router_test.js

```diff
@@ -25,42 +26,38 @@ QUnit.module("Ember Router", {
 
     // ensure rootURL is injected into any locations
     registry.injection('location', 'rootURL', '-location-setting:root-url');
-
-    Router = EmberRouter.extend();
   },
   teardown: function() {
     runDestroy(container);
-    registry = container = Router = router = null;
+    registry = container = null;
   }
 });
 
 test("can create a router without a container", function() {
-  Router.create();
+  createRouterWithContainer({ container: null });
 
   ok(true, 'no errors were thrown when creating without a container');
 });
 
 test("should not create a router.js instance upon init", function() {
-  createRouter();
+  var router = createRouterWithContainer();
 
   ok(!router.router);
 });
 
 test("should destroy its location upon destroying the routers container.", function() {
-  createRouter();
-
+  var router = createRouterWithContainer();
   var location = router.get('location');
 
-  run(container, 'destroy');
+  runDestroy(container);
 
   ok(location.isDestroyed, "location should be destroyed");
 });
 
 test("should instantiate its location with its `rootURL`", function() {
-  createRouter({
+  var router = createRouterWithContainer({
```

Comment:
i think we can leave this named as `createRouter` and just assume with  a container, unless container:null is passed